pulsar-broker/src/test/java/com/yahoo/pulsar/broker/service/BacklogQuotaManagerTest.java

```diff
@@ -77,7 +77,7 @@ void setup() throws Exception {
             bkEnsemble.start();
 
             // start pulsar service
-            config = create(new Properties(System.getProperties()));
+            config = create(new Properties(System.getProperties()), ServiceConfiguration.class);
```

Comment:
Should have a create that doesn't take any properties? In general we should try to avoid using `System.getProperties()` since it can lead to unpredictable tests, depending on what was passed on the command line.